app/utils.py

```diff
@@ -89,47 +101,31 @@ def choose_layer_style(n_clicks):
 
 # ----------------------------------------------------------------------------------------------------------------------
 # Information box
-# The following block is used to build the box in the top-right corner and fill it in with the relevant information.
-
-def build_info_box(feature=None):
-    """
-    This function creates the HTML components of the information box.
-
-    It takes as argument the polygon hovered by the user in the departments GeoJSON object, if any.
-
-    It returns:
-
-    - the title of the information box and the name of the hovered when relevant;
-    - the title of the information box and a generic message when there is no department being hovered.
-    """
-    header_dept = [html.H4('Département sélectionné :')]
+# The following block is used to build the map filters in the bottom left corner
 
-    if feature:
-        return header_dept + [html.B(feature['properties']['nom'])]
+def build_map_filters(feature=None):
```

Comment:
Is this function still necessary if it simply returns the output of another? Should we add something here?